build.gradle

```diff
@@ -93,19 +93,21 @@ subprojects {
         grpcNetty: "io.grpc:grpc-netty:${grpcVersion}",
         grpcStub: "io.grpc:grpc-stub:${grpcVersion}",
         grpcAuth: "io.grpc:grpc-auth:${grpcVersion}",
+        grpcProtobuf: "io.grpc:grpc-protobuf:${grpcVersion}",
         guava: 'com.google.guava:guava:18.0',
         jsr305: 'com.google.code.findbugs:jsr305:3.0.0',
         autovalue: 'com.google.auto.value:auto-value:1.2',
         joda: 'joda-time:joda-time:2.8.2',
         auth: 'com.google.auth:google-auth-library-oauth2-http:0.4.0',
-        commonprotos: "com.google.api.grpc:grpc-google-common-protos:${commonProtosVersion}",
+        commonprotos: "com.google.api.grpc:proto-google-common-protos:${commonProtosVersion}",
```

Comment:
The convention with the other dependencies is to use camel case, so this would be `commonProtos`